projects/aca-content/src/lib/components/layout/app-layout/app-layout.component.html

```diff
@@ -1,21 +1,14 @@
 <adf-upload-drag-area [rootFolderId]="currentFolderId" [disabled]="!canUpload">
   <adf-sidenav-layout
     #layout
-    [sidenavMin]="70"
-    [sidenavMax]="320"
+    [sidenavMin]="0"
+    [sidenavMax]="289"
     [stepOver]="600"
     [hideSidenav]="hideSidenav"
     [expandedSidenav]="expandedSidenav"
     (expanded)="onExpanded($event)"
   >
-    <adf-sidenav-layout-header>
-      <ng-template let-isMenuMinimized="isMenuMinimized">
-        <app-header role="heading" aria-level="1" *ngIf="!hideSidenav" (toggleClicked)="layout.toggleMenu()" [expandedSidenav]="!isMenuMinimized()">
```

Comment:
this removes the collapsible support for the sidebar, I think it should still be supported

Reply:
No, It's not supported. In the existing layout, collapsible is shown in the header, and in the new layout, collapse functionality is shown in the side navigation

Reply:
Once the sidenav and header changes are approved, I will raise a separate PR for collapse and expand functionality.

Reply:
you can add these changes to this PR, as without that some e2e tests may be failing and PR won't get merged